packages/redis/src/index.ts

```diff
@@ -131,6 +131,9 @@ export function redisStreamSource(configuration: IRedisInputStreamOptions): IInp
         base64Encode: true,
         consumerId: generate(),
         consumerGroupStartId: "$",
+        batchSize: 10,
+        blockTimeout: 1000,
```

Comment:
do 100 instead of 1000 ... 1000 would be enough to for CC to detect that graceful shutdown isn't working and then it will fallback to forceful shutdown

Reply:
Okay, I'll change this as long as you're okay with the ramification that redis will receive 2-3 calls every 100ms (xpending, xclaim, xreadgroup) 👍